### DISCUSSION TITLE
Set a ceiling for Nightscout on Google Cloud cost

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2455] [General]
I have seen this somewhere.  I would like to find it and also add it to the guide.
If you can help me find the answer, please post here.

**Comment by ireneusz-ptak** [comment]
https://cloud.google.com/billing/docs/how-to/budgets

But it only sends an email, we may as well set up a script which will disable the project billing if the quota is reached.
